cmd/yurthub/app/options/options.go

```diff
@@ -225,6 +227,8 @@ func (o *YurtHubOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.CoordinatorStorageAddr, "coordinator-storage-addr", o.CoordinatorStorageAddr, "Address of Yurt-Coordinator etcd, in the format host:port")
 	bindFlags(&o.LeaderElection, fs)
 	fs.BoolVar(&o.EnablePoolServiceTopology, "enable-pool-service-topology", o.EnablePoolServiceTopology, "enable service topology feature in the node pool.")
+	fs.StringVar(&o.CloudProviderIP, "cloud-provider-ip", o.CloudProviderIP, "the ip address of cloud provider that used for local mode yurthub.")
```

Comment:
the ip address of cloud provider that used for local mode yurthub. --> the address(ip:port) of host kubernetes cluster that used for yurthub local mode.

Reply:
done.